### PR TITLE
[AzureRT] Sprint 59-1

### PR DESCRIPTION
**Pull request by hyonholee**
New-AzureRmVmssIpConfig bug fix and test update.
Remove-AzureAvailabilitySet bug fix
Export-AzureVM and Import-AzureVM bug fix

### CODE REVIEW

File: src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/VirtualMachineScaleSetTests.ps1 [hunk 1]
```diff
@@ -389,26 +338,200 @@ function Test-VirtualMachineScaleSetReimageUpdate
...
+        # Validate Storage Profile
+        Assert-AreEqual 'test' $vmss.VirtualMachineProfile.StorageProfile.OsDisk.Name;
+        Assert-AreEqual 'FromImage' $vmss.VirtualMachineProfile.StorageProfile.OsDisk.CreateOption;
+        Assert-AreEqual 'None' $vmss.VirtualMachineProfile.StorageProfile.OsDisk.Caching;
+        Assert-AreEqual $vhdContainer $vmss.VirtualMachineProfile.StorageProfile.OsDisk.VhdContainers[0];
+        Assert-AreEqual $imgRef.Offer $vmss.VirtualMachineProfile.StorageProfile.ImageReference.Offer;
+        Assert-AreEqual $imgRef.Skus $vmss.VirtualMachineProfile.StorageProfile.ImageReference.Sku;
+        Assert-AreEqual $imgRef.Version $vmss.VirtualMachineProfile.StorageProfile.ImageReference.Version;
+        Assert-AreEqual $imgRef.PublisherName $vmss.VirtualMachineProfile.StorageProfile.ImageReference.Publisher;
+
```
Comment (DeepakRajendranMsft) [inline src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/VirtualMachineScaleSetTests.ps1]: Thanks for writing this detailed test case, I have a small request, could you please add these 2 lines here?

#validate Network Profile
 $expectedLb = Get-AzureRmLoadBalancer -Name $lbName -ResourceGroupName $rgname
Assert-AreEqual 2 $expectedLb.InboundNatRules.Count

**Comment by DeepakRajendranMsft** [comment]
LGTM

File: src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/VirtualMachineScaleSetTests.ps1 (hunk 1 above)
Comment (huangpf) [inline src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/VirtualMachineScaleSetTests.ps1]: @DeepakRajendranMsft 
Maybe you can add it in a separate follow-up PR? @hyonholee is OOF...

**Comment by huangpf** [comment]
LGTM.

File: src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/VirtualMachineScaleSetTests.ps1 (hunk 1 above)
Comment (DeepakRajendranMsft) [inline src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/VirtualMachineScaleSetTests.ps1]: @huangpf  sure that works!

**Comment by hovsepm** [comment]
I've re-targeted this PR to hotfix branch.

**Pull request closed without merging**